File: Readio/src/main/java/ninegle/Readio/library/dto/book/LibraryBookListResponseDto.java
package ninegle.Readio.library.dto.book;

import java.util.List;

import lombok.Builder;

@Builder
public record LibraryBookListResponseDto(LibraryDto libraryDto, List<AllLibraryBooksDto> allLibraryBooks,
										 long totalCount, long page, long size) {

}

File: Readio/src/main/java/ninegle/Readio/library/dto/book/LibraryBookListResponseDto.kt
package ninegle.Readio.library.dto.book


data class LibraryBookListResponseDto(
    val libraryDto: LibraryDto,
    val allLibraryBooks: List<AllLibraryBooksDto>,
    val totalCount: Long,
    val page: Long,
    val size: Long
)
